.circleci/workflows.yml

```diff
@@ -2738,6 +2760,12 @@ darwin-x64-workflow: &darwin-x64-workflow
         resource_class: macos.x86.medium.gen2
         requires:
           - darwin-x64-build
+    - server-unit-tests-cloud-environment:
+        name: darwin-x64-driver-server-unit-tests-cloud-environment
+        executor: mac
+        resource_class: macos.x86.medium.gen2
```

Comment:
not sure if this needs to be this big bc these are just unit tests

Reply:
Updated: [`e358a66` (#25837)](https://github.com/cypress-io/cypress/pull/25837/commits/e358a663e29e67a10a15cbd17f191d2f472d57b0)